data/data-pipeline/data_pipeline/etl/sources/census_acs/etl.py

```diff
@@ -361,22 +440,56 @@ def transform(self) -> None:
             1 - df[self.COLLEGE_ATTENDANCE_FIELD]
         )
 
-        # strip columns
-        df = df[self.COLUMNS_TO_KEEP]
+        # We preserve the null character of missing income even when college
+        # attendance is not null (otherwise this would be hard set to 0)
+        # and left-cut-off values at 0%
+        logger.info("Adjusting income information")
+        df[
```

Comment:
Note to self: I am wondering if it makes more sense to impute both raw income and raw college students, and then compute from what's imputed. Since the vast majority of tracts missing income are also missing college students, this won't be a particularly big shift, but may help smooth high college / low income places. 

Here, as a reminder, we make the assumption that 100% of college students are earning below 200FPL (or $34000). This is slightly below the median income of full time workers who have some college but no degree (https://nces.ed.gov/programs/coe/pdf/coe_cba.pdf)